backend/config/curation-api.yml

```diff
@@ -862,26 +978,30 @@ components:
           type: string
       type: object
     processing_status_collection:
-      description: >-
+      description: |
         Processing status for a Collection
-
-        * `"FAILURE"`: one or more Datasets ran into an error during processing. The CELLxGENE team will reach out to help you.
-        * `"PENDING"`: one or more Datasets are still being processed.
-        * `"SUCCESS"`: all Datasets have been processed successfully.
-        * `null`: the Collection has no Datasets.
+        | `processing_status` | Description |
+        | ------------------- | ----------- |
+        | `"FAILURE"` | one or more Datasets ran into an error during processing. More information can be found in the Dataset processing_status logs. |
+        | `"PENDING"` | one or more Datasets are still being processed. |
+        | `"SUCCESS"` | all Datasets have been processed successfully. |
+        | `null` | the Collection has no Datasets. |
       enum:
```

Comment:
Does this enum need to be updated?

Reply:
if you mean for `null` we don't need to specify that as an ENUM. It means the value is blank or not in the response.